### PR TITLE
Added Semantic HTML and Simple UI

### LINKED ISSUE #1: JSON Generator
Lacking CSS, UI, and mobile responsiveness. 

I would like to contribute a little on its UI and .css since this project has helped me greatly.

### PR DESCRIPTION
**Pull request by KishonShrill**
# Summary of Added changes
- This changes resolves #1 
- Full explanation will be found in contributor-logs > April 14, 2024.md

## HTML
- Used semantic code such as aside, main, and header to visually arrange the div's and inputs

## CSS
- Style tags is separated to an external stylesheet and imported to the html
- Added classes to style the Interface
- Made the website responsive for phone users.

## JavaScript
- Changed some variable names for readability and to target specific <div> parents.

Signed-off-by: Kishon Shrill <crystalbluew@gmail.com>

### CODE REVIEW

File: functions.js [hunk 1]
```diff
@@ -336,35 +352,33 @@ function saveNewData(id){
...
+			for (var j = 0; j < children[i].childNodes.length; j++) {
+				if (children[i].childNodes[j].className === "keyArea") {
 					//get key
-					if(i === 1){
-						key = (children[i].childNodes[j].childNodes[3]).value;
-					}else{
-						key = (children[i].childNodes[j].childNodes[1]).value;
-					}
-				}else if(children[i].childNodes[j].className === "value"){
+					console.log("Imong mama");
```
Comment (syncopika) [inline functions.js]: do you think we can remove this? 😆

**Comment by syncopika** [comment]
These changes look fantastic 🤩, thank you so much! 🙏 

I really love the documentation as well! Just curious - did you write all that out by hand (specifically the contributor-logs file) or was there some automation around it?

just for future reference:
old UI:
image

new UI with this PR 🚀 :
image

File: functions.js (hunk 1 above)
Comment (KishonShrill) [reply functions.js]: ## Redundant console. Logs are now removed. 

Console.log was used to determine whether the function was accessed and would display "Imong mama" (means "your mom" in our country 🇵🇭) when the if-statement passed. I used it as a flag to check whether the statements had a bug or not.

**Comment by KishonShrill** [comment]
All the .css design was written by hand. I had a little help with AI when explaining the documentation and finding JavaScript bugs 👍

**Pull request merged**